### PR TITLE
Move bulk video downloads to a celery task

### PR DESCRIPTION
**Pull request by cwisecarver**
- Requesting to download multiple attachments starts a celery task that generates a zip file, uploads it to google cloud storage, creates a signed url, and emails it to the requesting user.
    - If a zip file with the same videos already exists on google cloud storage it just generates the link and emails it.

### CODE REVIEW

**Comment by cwisecarver** [comment]
Hey @pattisdr I think I'm done with this. If you want to functionally test it you'll need credentials for AWS and Google Cloud Storage in your environment variables. You'll also need to set the env var for the GCS private bucket. If you want to do this lmk and I'll send you creds in a super secret way.

The only thing I'd be worried about is if the matching/consent stuff worked. I tested it pretty well but it couldn't hurt to check it again.

File: attachment_helpers.py [hunk 1]
```diff
@@ -29,3 +29,14 @@ def get_download_url(video_key):
     """
     s3Client = boto3.client('s3')
     return s3Client.generate_presigned_url('get_object', Params = {'Bucket': settings.BUCKET_NAME, 'Key': video_key}, ExpiresIn = 60)
+
+
+def get_study_attachments(study, orderby, match):
```
[the remark this answers is not in the document]
Comment (cwisecarver) [reply attachment_helpers.py]: 👍 Done. Waiting for tests then merging.

**Comment by pattisdr** [comment]
Looks good to me, nice work on this!!

**Comment by icereval** [comment]
:+1: :tada:

**Pull request merged**